src/test/java/TestEAN/testEan.java

```diff
@@ -0,0 +1,42 @@
+package TestEAN;
+
+import org.fundacionjala.coding.EAN.EAN;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Esta es la clase Test que hace correr las pruebas de la clase EAN.
+ */
+
+public class testEan {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00101)